guide/creating-your-bot/command-deployment.md

```diff
@@ -0,0 +1,126 @@
+# Registering slash commands
+
+::: tip
+This page assumes you use the same file structure as our [Slash commands](/slash-commands/) section, and the scripts provides are made to function with that setup. Please carefully read those first so that you can understand the methods used in this section.
+
+If you already have slash commands set up and deployed for your application and want to learn how to respond to them, refer to the following section on [replying to slash commands](/slash-commands/replying.md).
+:::
+
+In this section, we'll cover how to register your commands to Discord using discord.js!
+
+## Command registration
+
+Slash commands can be registered in two ways; in one specific guild, or for every guild the bot is in. We're going to look at single-guild registration first, as this is a good way to develop and test your commands before a global deployment.
```

Comment:
Inconsistent use of server and guild (which is probably a larger point stretching across all of the guide).
I think we should either

- Stick to the same phrasing throughout (guild or server) the explanatory text
- If using guild, explain it at the first occurrence (be that in code or text)

Reply:
Have addressed this with a tip box in main-file.md as this is where we first refer to "guilds" (when referencing the intents).